lib/src/services/databases/calendar/hybrid.dart

```diff
@@ -38,6 +38,9 @@ class HybridCalendar extends HybridDatabase<CalendarInterface> implements Calend
 	@override
 	Future<List<Map>> getSchedules() => local.getSchedules();
 
+	@override
+	Future<Map> getDefaultSchedules() => local.getDefaultSchedules();
```

Comment:
```suggestion
	Future<Map<String, String>> getDefaultSchedules() => local.getDefaultSchedules();
```

Be sure to include types _and_ type arguments